cmd/sysl/Codegen_test.go

```diff
@@ -17,8 +17,8 @@ import (
 func TestGenerateCode(t *testing.T) {
 	t.Parallel()
 
-	output, err := GenerateCodeWithParams(".", "tests/model.sysl", ".", "tests/test.gen.sysl",
-		"tests/test.gen.g", "javaFile")
+	output, err := GenerateCodeWithParams(testDir, "model.sysl", testDir, "test.gen.sysl",
+		testDir+"test.gen.g", "javaFile")
```

Comment:
instead of concatenating strings, use `filepath.Join` so Windows won't get unix paths and also do a `fielpath.Clean` on `testDir` to convert it into os specific paths. Changing this might fix the failing Windows CI.

Reply:
I don't like filepath.join; syntactically it makes it a lot more to look at for something that is really simple
looking at varName + "string" is easier than looking at filepath.join(something, something else)

Ideally the whole test suite would be restructured so that testDir didn't need to be used at all, but until then I'm happy with testDir + "filename"

Reply:
oh, to be honest I didn't even think about windows filesystems